src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -14,6 +14,7 @@ import RunwayCollection from './runway/RunwayCollection';
 import StaticPositionModel from '../base/StaticPositionModel';
 import { isValidGpsCoordinatePair } from '../base/positionModelHelpers';
 import { degreesToRadians, parseElevation } from '../utilities/unitConverters';
+import { getNewWind } from '../utilities/windUtilities';
```

Comment:
avoid using the term `get` with methods or functions.  you will see it used sparingly, but should be used only to denote a getter type method.

in this case, the word `calculate` would be more appropriate.